### PR TITLE
add mobilenet_v2 support

### PR DESCRIPTION
**Pull request by hellowaywewe**

### CODE REVIEW

File: tinyms/utils/train/lr_generator.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        else:
+            lr = lr_end + \
+                 (lr_max - lr_end) * \
+                 (1. + math.cos(math.pi * (i - warmup_steps) / (total_steps - warmup_steps))) / 2.
+        if lr < 0.0:
+            lr = 0.0
+        lr_each_step.append(lr)
+
+    current_step = global_step
+    lr_each_step = np.array(lr_each_step).astype(np.float32)
```
Comment (leonwanghui) [inline tinyms/utils/train/lr_generator.py:52]: This is better to change to:
```python
import tinyms as ts

lr_each_step = ts.array(lr_each_step, dtype=ts.float32)
```

File: tests/st/mobilenetv2.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    # define the loss function
+    if args_opt.label_smooth > 0:
+        loss = CrossEntropyWithLabelSmooth(smooth_factor=args_opt.label_smooth,
+                                           num_classes=args_opt.num_classes)
+    else:
+        loss = SoftmaxCrossEntropyWithLogits(sparse=True, reduction='mean')
+
+    # get learning rate
+    step_size = ds_train.get_dataset_size()
+    lr = Tensor(mobilenetv2_lr(global_step=0, lr_init=.0, lr_end=.0, lr_max=0.8, warmup_epochs=0,
```
Comment (leonwanghui) [inline tests/st/mobilenetv2.py:100]: The `Tensor` wrapper can be removed after modify `mobilenetv2_lr` method.

File: tests/st/mobilenetv2.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                                            repeat_size=repeat_size,
+                                            batch_size=batch_size,
+                                            training=training)
+
+    return cifar10_ds
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='MobileNetV2 Image classification')
+    parser.add_argument('--device_target', type=str, default="GPU", choices=['Ascend', 'GPU', 'CPU'],
```
Comment (leonwanghui) [inline tests/st/mobilenetv2.py:57]: default should be `CPU`

File: tinyms/callbacks.py [hunk 4]
```diff
@@ -13,8 +13,66 @@
...
 # ============================================================================
 
+
+import time
+import tinyms as ts
 from mindspore.train import callback
 from mindspore.train.callback import *
 
+
+class LossTimeMonitor(Callback):
```
Comment (leonwanghui) [inline tinyms/callbacks.py:23]: This class should be placed after `__all__`

File: tinyms/utils/train/cb_config.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+
+from tinyms.callbacks import ModelCheckpoint, CheckpointConfig, LossTimeMonitor
+
+
+def mobilenetv2_cb(device_target, lr, is_saving_checkpoint, save_checkpoint_epochs, step_size):
```
Comment (leonwanghui) [inline tinyms/utils/train/cb_config.py:19]: This is better to move to `callbacks.py`.

**Review by leonwanghui — APPROVED** [review]
LGTM

**Review by huxiaoman7 — APPROVED** [review]

**Pull request merged**